src/App.jsx

```diff
@@ -103,6 +157,19 @@ function App() {
             />
```

Comment:
⚖️ **Midlevel Developer**

The conditional rendering for the "Expected Years in Retirement" field (lines 157-170) is a clever touch, but you should probably disable or hide the calculate button if the field is empty or invalid for 1/N withdrawals. At my last company, empty fields caused so many support calls. Defensive UI and input validation are best practices!